src/Apps/Juno/Command.hs

```diff
@@ -19,17 +19,22 @@ import Data.Aeson (encode, object, (.=))
 import Data.Aeson.Encode.Pretty (encodePretty)
 
 import Juno.Types (CommandEntry(..), CommandResult(..))
+import Juno.Types.Base (RequestId)
 import Schwifty.Swift.M105.Types (SWIFT)
 
 import Apps.Juno.Parser
 import Apps.Juno.Ledger (runQuery, convertQuery)
 
 -- state of hopper
-newtype JunoEnv = JunoEnv {getStateMVar :: MV.MVar (DEval.PersistentState, Map.Map DEval.TransactionId SWIFT)}
+newtype JunoEnv = JunoEnv {getStateMVar :: MV.MVar (DEval.PersistentState
```

Comment:
arrgh really hating our use of tuples inside of an mvar. Should really be a type, with lenses to ease the garbage code in `runCommand`